[API/compareTest] Return result when files do not exist

anyj0527 opened the pull request:
- When the file does not  exist, in BusyBox condition, the bufsize is null and results of `dd` are same error message. And unexpected PASS result follows.
- This patch prevents this unexpected behavior.

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

anyj0527 commented:
This resolves https://github.com/nnstreamer/meta-neural-network/issues/54

myungjoo commented on ssat-api.sh:
```diff
@@ -233,50 +233,54 @@ function callTestExitEq() {
-			cmp -n `${StatCmd_GetSize} $2` $1 $2
-			output=$?
+	if [[ ! -f "$1" || ! -f "$2" ]]; then
+		testResult $output "$3" "$4" $6
```
please do return in ```then``` case so that you don't need to add ```else``` case and indent further. Let's minimize the changes.

wooksong replied:
Agreed with https://github.com/myungjoo/SSAT/pull/15/files#r501473366.

@myungjoo Q: Which repository is the upstream for SSAT? github or git.tzien.org?

anyj0527 replied:
Updated, please check!

myungjoo replied:
github.com/myungjoo/SSAT is upstream and others are forks.

myungjoo replied:
Anyway, don't do SR until 6.0 M2 is released.

anyj0527 committed:
[API/compareTest] Return result when files do not exist

- When the file does not  exist, in BusyBox condition, the bufsize is null and results of `dd` are same error message. And unexpected PASS result follows.
- This patch prevents this unexpected behavior.

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>
